.circleci/config.yml

```diff
@@ -58,9 +53,10 @@ jobs:
       - restore_cache:
           key: v2-dependencies-{{ checksum "yarn.lock" }}
       - restore_cache:
-          key: vanilla-build-{{ .Revision }}
-      - restore_cache:
-          key: react-build-{{ .Revision }}
+          key: package-build-{{ .Revision }}
+      - run:
+          name: Rerun yarn to re-link peer packages
+          command: yarn
```

Comment:
I'm not 100% certain this line is necessary. This stuff should be cached already.

Reply:
✅ We just need to cache dependencies that Circle expects to use. I added storybook/node_modules to the cache for this reason.